src/server.js

```diff
@@ -85,6 +85,7 @@ const plugin: RPCPluginType =
     deps: {
       emitter: UniversalEventsToken,
       handlers: RPCHandlersToken,
+      bodyParserOptions: BodyParserOptionsToken,
```

Comment:
We should make this optional. If you change it to `BodyParserOptionsToken.optional` it will do the trick

Reply:
It would be great if we could get a test case also

Reply:
updated.